src/docs/working-with-actors/Actor lifecycle.md

```diff
@@ -1,67 +1,111 @@
 ---
 layout: docs.hbs
-title: Actor lifecycle
+title: Actor API
 ---
+# Actor API
```

Comment:
This is a good section but I don't think it flows with the rest of the documentation. From a users perspective, it seems to be mashing two concepts together. Maybe move it to a separate document or move it towards the end of this one?